src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -60,6 +65,7 @@ export class HoldConfirmation extends React.Component {
             {blocked}
             {ptypeDisallowsHolds}
             {defaultText}
+            {reachedHoldLimit}
```

Comment:
I don't think it actually makes a difference but I'd move `reachedHoldLimit` above `defaultText`